dapp/views/transactions/transactions_list/transactions_list.jsx

```diff
@@ -1,96 +1,54 @@
-import React                               from 'react'
-import { bindActionCreators }              from 'redux'
-import { connect }                         from 'react-redux'
-import Transaction                         from './transaction'
-import { Preloader, Bouncing }             from '@poplocker/react-ui'
-import { updateHistory, upPage, downPage } from 'lib/store/actions'
+import React                   from 'react'
+import { connect }             from 'react-redux'
+import Transaction             from './transaction'
+import { Preloader, Bouncing } from '@poplocker/react-ui'
+import { rpc }                 from 'lib/rpc_calls'
 
 import './transactions_list.css'
 
 class TransactionList extends React.Component {
-  componentDidMount() {
-    this.fetchHistory(1).then(this.props.updateHistory);
+  constructor(props) {
+    super(props);
+    this.listAddress = null;
   }
 
-  fetchHistory(page) {
-    return fetch(
-      `${process.env.ETHERSCAN_URL}&address=${this.props.address}&page=${page}`
-    ).then(response => response.json())
-     .then(response => response.result);
+  componentWillMount() {
+    this.listAddress = this.props.address;
```

Comment:
Moved to constructor to avoid using `componentWillMount()`
`listAddress` was introduced to fix an issue where if the address changed (i.e. from the PopLocker API) then the entire tx history would update because of the address change (albeit momentarily before a new history is fetched). The way to fix this is to pass a copy of the address to the child txs.